travis.sh

```diff
@@ -3,6 +3,9 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+CTEST_OUTPUT_ON_FAILURE=ON
+export CTEST_OUTPUT_ON_FAILURE
```

Comment:
Fair enough. Wasn't aware that Dash (`/bin/sh` in most Linux distributions, as opposed to Bash in OSX) did not support that.